### ISSUE TITLE
Opportunity to eliminate redundant HTML file

### ISSUE THREAD

**Issue by Slapbox** [reZach/secure-electron-template#52]
I've been toying with this excellent template and moving our own work to more closely replicate yours.

While looking for a way to make this compatible with the Webpack DLL plugins I was looking through the HTML Webpack plugin's options and noticed there's an option to inject the `<base />` tag.

https://github.com/jantimon/html-webpack-plugin#base-tag

PS: Any thoughts on DLL compatibility? I'm not sure this is still relevant on Webpack 5 so I didn't want to make an issue, but we're using Webpack 4 and I don't see any way to make it work so far.

Thanks again for the share!

**Comment by reZach** [comment]
Thank you for the compliment, @Slapbox.

In my minute or two of toying with this, I couldn't get it to work, but I expect changes need to be made _at least_ [here](https://github.com/reZach/secure-electron-template/blob/master/webpack.production.js#L13) and [here](https://github.com/reZach/secure-electron-template/blob/master/app/electron/main.js#L97). It might need to be fiddled with because webpack might run _after_ electron tries to load the HTML file for the app - something you can look further into?

I expect that's the root problem that bears more investigation.
I imagine you only want one index file so you only have to make updates in one spot?

**Comment by Slapbox** [comment]
Hey @reZach thanks for your speedy reply!

When I made this issue I hadn't yet gotten the `app://rse` protocol working.

I got that working this hour and managed to get injecting `base` working as expected too, but I'm moving our existing codebase towards the best practices of Secure Electron Template, so there are some significant differences in our setup

The most notable differences are that we're still on Webpack@4 and Electron@8.

I'm going to keep on working towards getting us on Webpack@5 and Electron@11 and I'll loop back once we get there and report on whether or not this is still working for us.

Thanks again for sharing your work, it's really excellent!
